react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -206,8 +214,11 @@ class OverflowMenu extends PureComponent<Props, State> {
  * @returns {Props}
  */
 function _mapStateToProps(state) {
+    const { disableSelfView } = state['features/base/settings'];
+
     return {
         _bottomSheetStyles: ColorSchemeRegistry.get(state, 'BottomSheet'),
+        _disableSelfView: Boolean(disableSelfView),
```

Comment:
nit: rename to _selfViewHidden for more clarity, as we generally use _disabled for features that are actually not available at all.